src/plugins/auto_batch/src/plugin.cpp

```diff
@@ -8,38 +8,35 @@
 
 #include "compiled_model.hpp"
 #include "dimension_tracker.hpp"
-#include "ie_icore.hpp"
-#include "ie_metric_helpers.hpp"
-#include "ie_ngraph_utils.hpp"
-#include "ie_performance_hints.hpp"
 #include "openvino/pass/manager.hpp"
 #include "openvino/runtime/intel_gpu/properties.hpp"
 #include "openvino/util/common_util.hpp"
 #include "transformations/common_optimizations/dimension_tracking.hpp"
 #include "transformations/init_node_info.hpp"
 #include "transformations/utils/utils.hpp"
+OPENVINO_SUPPRESS_DEPRECATED_START
+#include "ie_ngraph_utils.hpp"
+OPENVINO_SUPPRESS_DEPRECATED_END
```

Comment:
Why not remove these 3 lines?

Reply:
Need at: https://github.com/openvinotoolkit/openvino/blob/6202221b933e2f58c3ae60278679a511841d654d/src/plugins/auto_batch/src/plugin.cpp#L317
Here need the rt info for model convert.